pkg/e2e/compose_build_test.go

```diff
@@ -26,7 +27,18 @@ import (
 	"gotest.tools/v3/icmd"
 )
 
-func TestLocalComposeBuild(t *testing.T) {
+func TestLocalComposeBuild_Classic(t *testing.T) {
+	os.Setenv("DOCKER_BUILDKIT", "0")    //nolint:errcheck
```

Comment:
nit: here we could either do
```suggestion
	_ = os.Setenv("DOCKER_BUILDKIT", "0")
```
or check the error?

Reply:
sorry, was not intended to be merged, just wanted to do a quick and dirty try to get both builder classic and buildkit CI run